src/Summoner/Template.hs

```diff
@@ -158,16 +198,21 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
     createCabalBenchmark :: Text -> Text
     createCabalBenchmark r =
         [text|
-        benchmark ${repo}-benchmark
-          type:                exitcode-stdio-1.0
-          default-language:    Haskell2010
-          ghc-options:         -Wall -Werror -O2 -threaded -rtsopts -with-rtsopts=-N
-          hs-source-dirs:      benchmark
-          main-is:             Main.hs
-          build-depends:       $base
-                             , gauge
-                             $customPreludePack
-                             $r
+        benchmark             ${repo}-benchmark
+          type:               exitcode-stdio-1.0
+          default-language:   Haskell2010
+          ghc-options:        -Wall
+                              -threaded
+                              -rtsopts
+                              -with-rtsopts=-N
+                              -02
+                              $ghcOptions
+          hs-source-dirs:     benchmark
+          main-is:            Main.hs
```

Comment:
Let's unify structure of stanzas. And let's have `hs-source-dirs` with `main-is` before `ghc-options` field how it's already done in executable an test-suite.